### ISSUE TITLE
VMPing crashing when Notifications are set to When Minimized

### ISSUE THREAD

**Issue by chuckbales** [R-Smith/vmPing#36]
I've noticed that when i have the notification option set to When Minimized instead of Always, vmping will consistently crash. It doesn't always crash after the same amount of time, but it will always crash. I can leave it going with a dozen pings all day long when notification is set to Always and have no issue, but when set to When Minimized it seems to crash within 10 minutes (sometimes within a few seconds).

I've tried with the latest 1.3.3 on both my Win 10 laptop and a Server 2019 instance, they both crash with these events in the Event Viewer

```
Log Name:      Application
Source:        .NET Runtime
Date:          4/22/2020 11:51:47 AM
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      server-name-here
Description:
Application: vmPing.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.InvalidOperationException
   at System.Windows.Threading.Dispatcher.VerifyAccess()
   at vmPing.Classes.Probe.TriggerStatusChange(vmPing.Classes.StatusChangeLog)
   at vmPing.Classes.Probe+<PerformIcmpProbe>d__1.MoveNext()
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

```
Log Name:      Application
Source:        Application Error
Date:          4/22/2020 11:51:47 AM
Event ID:      1000
Task Category: (100)
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      server-name-here
Description:
Faulting application name: vmPing.exe, version: 1.3.3.0, time stamp: 0x5de8212f
Faulting module name: KERNELBASE.dll, version: 10.0.17763.1098, time stamp: 0xc5222078
Exception code: 0xe0434352
Fault offset: 0x0000000000039179
Faulting process id: 0x2298
Faulting application start time: 0x01d618bde41152fa
Faulting application path: C:\Users\username\Downloads\vmPing.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: 571abba5-c6c0-454d-8d60-3ea0cc978b05
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by R-Smith** [comment]
Thank you for the report.  I was able to re-create the crash and I'll have a fix out soon.

**Comment by R-Smith** [comment]
Fixed here 680a0057095555713a74a9a1cd2a3236d3a01181

Available in release 1.3.4.
